### PR TITLE
add auth controller

### PR DESCRIPTION
**Pull request by huuanh99**
## Related Tickets:
- [#43726](https://edu-redmine.sun-asterisk.vn/issues/43726)

## WHAT this PR does?
create auth controller

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by huuanh99** [comment]
ready

File: app/Http/Controllers/Auth/AuthenticatedSessionController.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+use Illuminate\Support\Facades\Auth;
+
+class AuthenticatedSessionController extends Controller
+{
+    /**
+     * Display the login view.
+     *
+     * @return \Illuminate\View\View
+     */
+    public function create()
```
Comment (huyenttm-0901) [inline app/Http/Controllers/Auth/AuthenticatedSessionController.php]: sao `Display the login view.` mà em đặt tên function là `create` nhỉ?

File: app/Http/Controllers/Auth/RegisteredUserController.php [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+use Illuminate\Validation\Rules;
+
+class RegisteredUserController extends Controller
+{
+    /**
+     * Display the registration view.
+     *
+     * @return \Illuminate\View\View
+     */
+    public function create()
```
Comment (huyenttm-0901) [inline app/Http/Controllers/Auth/RegisteredUserController.php]: tương tự

**Comment by huuanh99** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**